api/rest_client/api_usage_examples.rest

```diff
@@ -1,76 +1,77 @@
 # This file contains example HTTP calls and it also the configuration
 # file for the VSCode plugin "REST Client" https://open-vsx.org/extension/humao/rest-client. 
 
-@API_SERVER = http://localhost:5001
-
-# Bearer Token starts with 'ey...', no quotes
-@AUTH_BEARER_TOKEN = ey...
-
+#Add the following to the .env file in this directory:
```

Comment:
It would be nice to move this documentation to the `README` and a bit more context about what purpose this client severs. 

And there is still a tiny bit of legacy documentation in the README, see the `Test-driven development` section.

Reply:
thanks for flagging, John. I agree that a README would be great to have since this is in its own folder. I know that I'm in the habit of looking for READMEs since we've been better about adding them. The top of this file looks very copy/pastable. could you update @frankhereford?

Reply:
Great idea, and yes, of course - it's moved and filled out a little bit too. Thank you. https://github.com/cityofaustin/atd-moped/pull/1614/commits/9a6b88423397d7f1d68398b188a67a5d6218de5c